diff -up QtAV-master/CMakeLists.txt.2~ QtAV-master/CMakeLists.txt
--- QtAV-master/CMakeLists.txt.2~	2023-12-10 23:49:37.396095964 +0100
+++ QtAV-master/CMakeLists.txt	2023-12-10 23:50:50.522967779 +0100
@@ -72,10 +72,10 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON) # for
 # Instruct CMake to run moc automatically when needed.
 set(CMAKE_AUTOMOC ON)
 if(NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-  set(QTAV_INSTALL_HEADERS ${CMAKE_INSTALL_PREFIX}/include)
-  set(QTAV_INSTALL_LIBS ${CMAKE_INSTALL_PREFIX}/lib)
-  set(QTAV_INSTALL_BINS ${CMAKE_INSTALL_PREFIX}/bin)
-  set(QTAV_INSTALL_QML ${QTAV_INSTALL_LIBS}/qml)
+  set(QTAV_INSTALL_HEADERS ${CMAKE_INSTALL_PREFIX}/include CACHE PATH "header install path")
+  set(QTAV_INSTALL_LIBS ${CMAKE_INSTALL_PREFIX}/lib CACHE PATH "library install path")
+  set(QTAV_INSTALL_BINS ${CMAKE_INSTALL_PREFIX}/bin CACHE PATH "binary install path")
+  set(QTAV_INSTALL_QML ${CMAKE_INSTALL_PREFIX}/qml CACHE PATH "qml install path")
 else()
   set(CMAKE_INSTALL_PREFIX ${QT_INSTALL_PREFIX} CACHE PATH "default install path" FORCE)
   set(QTAV_INSTALL_HEADERS ${QT_INSTALL_HEADERS})
